scoop-basics/src/main/java/com/example/scoop/basics/ui/transitions/dialogtransitions/screen/Dialog.java

```diff
@@ -9,10 +12,31 @@
 import com.lyft.scoop.ExitTransition;
 import com.lyft.scoop.Screen;
 import com.lyft.scoop.dagger.DaggerModule;
+import java.io.ByteArrayOutputStream;
 
 @Controller(DialogController.class)
 @DaggerModule(DialogModule.class)
 @EnterTransition(SlideUpTransition.class)
 @ExitTransition(SlideDownTransition.class)
 public class Dialog extends Screen {
+    private byte[] backgroundBitmap;
+
+    public void setBackground(BitmapDrawable backgroundDrawable) {
```

Comment:
The dialog controller is the one calling it right now as it's the one responsible for storing/restoring state. The transition is only responsible for capturing the state before the transition happens and sets the `to` view accordingly. I considered moving it to UIContainer, but this seems too specific of a view property (`BitmapDrawable` background) to store/restore on a framework level.